src/Commands/ChurnCommand.php

```diff
@@ -77,13 +79,22 @@ class ChurnCommand extends Command
     public function __construct()
     {
         parent::__construct();
-        $this->config = new Config(Yaml::parse(@file_get_contents(getcwd() . '/churn.yml')) ?? []);
+        $this->setConfig(new Config(Yaml::parse(@file_get_contents(getcwd() . '/churn.yml')) ?? []));
         $this->fileManager = new FileManager($this->config);
         $this->processFactory = new ProcessFactory($this->config);
         $this->resultsParser = new ResultsParser;
     }
 
     /**
+     * Config setter.
+     * @param Config $config
+     */
+    public function setConfig(Config $config)
```

Comment:
I'm not sure having a public method to set the config just to make the class testable is a worth it. I definitely would want this class testable but having a public method to set the config just doesn't seem right to me.

Can you think of any other ways to do this?

Reply:
ye trying to figure it out.